Decisions/scs-XXXX-v1-requirements-for-container-registry.md

```diff
@@ -0,0 +1,372 @@
+---
+title: Requirements for container registry
+type: Decision Record
+status: Draft
+track: KaaS
+---
+
+# Introduction  
+
+A container registry is a repository or collection of repositories that store and
+access container images. It acts as the intermediary for sharing container images
+between systems and could be connected directly to container orchestration platforms
+like Kubernetes.
```

Comment:
Continuation of https://github.com/SovereignCloudStack/standards/pull/212#discussion_r1104263337

Thanks for tackling the issue of Docker being included in this context!

I stumbled across the "connected directly to container orchestration platforms" wording again and I think it could be improved. It is not clear what this wording really means.

Another attempt could look like this:

```
A container registry is an infrastructure service to enable storing and accessing container
images. Images can be pushed to the registry by e. g. Continuous integration pipelines and
be pulled from by runtime environments like Kubernetes clusters.
```

Reply:
fixed 3b735e8aebd689adcfde00c9aabc8c25cd7816bb